qiskit/visualization/state_visualization.py

```diff
@@ -260,7 +260,7 @@ def plot_bloch_multivector(
         MissingOptionalLibraryError: Requires matplotlib.
         VisualizationError: if input is not a valid N-qubit state.
 
-    Example:
+    Example - bell state:
```

Comment:
Given that the previous example doesn't work at all, it would be better if we get rid of it completely out of the documentation.